.coafile

```diff
@@ -20,3 +20,7 @@ default_actions = SpaceConsistencyBear: ApplyPatchAction
 bears = GitCommitBear
 shortlog_trailing_period = False
 shortlog_regex = ([^:]*|\S+: [A-Z0-9*].*)
+
+[yml]
+bears = YAMLLintBears
```

Comment:
You can do modifications to commit, and in this PR only. Please have a look on the process here http://api.coala.io/en/latest/Developers/Git_Basics.html#follow-up  :) 👍